client/orderbook/orderbook.go

```diff
@@ -44,23 +54,30 @@ type cachedOrderNote struct {
 
 // OrderBook represents a client tracked order book.
 type OrderBook struct {
-	log          dex.Logger
-	seqMtx       sync.Mutex
-	seq          uint64
-	marketID     string
-	noteQueue    []*cachedOrderNote
+	log      dex.Logger
+	seqMtx   sync.Mutex
+	seq      uint64
+	marketID string
+
 	noteQueueMtx sync.Mutex
-	orders       map[order.OrderID]*Order
-	ordersMtx    sync.Mutex
-	buys         *bookSide
-	sells        *bookSide
-	syncedMtx    sync.Mutex
-	synced       bool
+	noteQueue    []*cachedOrderNote
+
+	ordersMtx sync.Mutex
+	orders    map[order.OrderID]*Order
+
+	buys  *bookSide
+	sells *bookSide
+
+	syncedMtx sync.Mutex
+	synced    bool
 
 	epochMtx     sync.Mutex
 	currentEpoch uint64
 	proofedEpoch uint64
 	epochQueues  map[uint64]*EpochQueue
+
+	baseFeeRate  uint64
+	quoteFeeRate uint64
```

Comment:
Will need these as the first OrderBook fields because of the sync/atomic bug on 32-bit platforms.